### PR TITLE
Guide

### PR DESCRIPTION
**Pull request by AntonNazarenko**

### CODE REVIEW

**Comment by robertkowalski** [comment]
it does not start:

```
Compiled with warnings.

./src/ui/Markdown/index.js
  Line 3:40:   Extra semicolon                                     semi
  Line 4:45:   Extra semicolon                                     semi
  Line 23:47:  Extra semicolon                                     semi
  Line 29:43:  Extra semicolon                                     semi
  Line 53:11:  Dangerous property 'dangerouslySetInnerHTML' found  react/no-danger

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
```

**Comment by robertkowalski** [comment]
had to rebase your PR against latest master

**Comment by robertkowalski** [comment]
after closing the guide in the middle there is a border around the element, can we remove it?

Screen Shot 2020-09-09 at 12 16 08

**Comment by robertkowalski** [comment]
Can we rename "Last" to "Finish" in the last step? Because clicking on "Last" closes the guide.

**Comment by robertkowalski** [comment]
Screen Shot 2020-09-09 at 12 19 10

File: src/pages/StrategyEditor/StrategyEditor.js [hunk 1]
```diff
@@ -20,6 +21,20 @@ export default class StrategyEditorPage extends React.Component {
...
     indicators: [],
     strategyContent: null,
+    steps: [
+      {
+        target: '.hfui-create-strategy__btn',
+        content: 'Create your own strategies.',
+      },
+      {
+        target: '.hfui-open-strategy__btn',
+        content: 'Or open existing one.',
```
Comment (robertkowalski) [inline src/pages/StrategyEditor/StrategyEditor.js:31]: `Or open an existing one`

File: src/pages/StrategyEditor/StrategyEditor.js [hunk 2]
```diff
@@ -20,6 +21,20 @@ export default class StrategyEditorPage extends React.Component {
...
+      },
+      {
+        target: '.hfui-markdown__wrapper',
+        content: 'here you can find all documentation you need, to code yourn own strategies.',
```
Comment (robertkowalski) [inline src/pages/StrategyEditor/StrategyEditor.js:35]: `yourn ` -> `your`

Comment (robertkowalski) [reply src/pages/StrategyEditor/StrategyEditor.js:35]: is this text final?

Maybe we can go with: `In this section you find the available function declarations to code your own strategies`

File: src/pages/Settings/Settings.js [hunk 3]
```diff
@@ -35,6 +37,32 @@ export default class Settings extends React.Component {
...
+          target: '.api-key',
+          content: 'In this field you should provide your API key to your bitfinex account.',
+        },
+        {
+          target: '.api-secret',
+          content: 'And here you should provide your secret key.',
+        },
+        {
+          target: '.settings-save',
+          content: 'After all steps press this button, to update your settings.',
```
Comment (robertkowalski) [inline src/pages/Settings/Settings.js:63]: not sure if we need a guide for this, those sections should be self-explanatory, right? 

i.e. `With this button you can enable/disable DMS switch.` does not provide much worth to the user

Maybe remove the guide from this page, and if required add some more context to the UI fields, example:

Screen Shot 2020-09-09 at 12 25 02

Comment (robertkowalski) [reply src/pages/Settings/Settings.js:63]: Screen Shot 2020-09-09 at 12 27 44

Comment (robertkowalski) [reply src/pages/Settings/Settings.js:63]: That could be done in a next PR...

File: src/components/GridLayoutPage/GridLayoutPage.js [hunk 4]
```diff
@@ -37,6 +38,28 @@ export default class GridLayoutPage extends React.Component {
     layoutDirty: false,
     addLayoutModalOpen: false,
     addComponentModalOpen: false,
+    steps: [
+      {
+        target: '.hfui-button',
+        content: 'Create new component.',
```
Comment (robertkowalski) [inline src/components/GridLayoutPage/GridLayoutPage.js:44]: maybe removing the `.` makes sense? for the guide-bubbles it looks nicer without the fullstop

File: src/pages/Trading/Trading.js [hunk 5]
```diff
@@ -20,7 +21,34 @@ const orderDefinitions = {
...
-
+  state = {
+    steps: [
+      {
+        target: '.icon-plus',
+        content: 'By clicking on this button you can add a new components to this page.',
+      },
+      {
+        target: '.icon-save',
+        content: 'You can save your layout, after you have customized it.',
```
Comment (robertkowalski) [inline src/pages/Trading/Trading.js:32]: `You can save your layout after you have customized it`

File: src/pages/Trading/Trading.js [hunk 6]
```diff
@@ -20,7 +21,34 @@ const orderDefinitions = {
 export default class Trading extends React.PureComponent {
   static propTypes = propTypes
   static defaultProps = defaultProps
-
+  state = {
+    steps: [
+      {
+        target: '.icon-plus',
+        content: 'By clicking on this button you can add a new components to this page.',
```
Comment (robertkowalski) [inline src/pages/Trading/Trading.js:28]: `component`

File: src/pages/Trading/Trading.js [hunk 7]
```diff
@@ -20,7 +21,34 @@ const orderDefinitions = {
...
+        target: '.hfui-orderformmenu__wrapper',
+        content: 'Here you can find all orders you need.',
+      },
+      {
+        target: '.hfui-statusbar__wrapper',
+        content: 'This bar shows all statuses you need to know.',
+      },
+      {
+        target: '.hfui-statusbar__left',
+        content: 'Version of your app, there will be "Update" button, if you are running old version.',
```
Comment (robertkowalski) [inline src/pages/Trading/Trading.js:48]: `Here you can find the current version of your app, in case there is an update, there will be an update button`

**Review by robertkowalski — CHANGES_REQUESTED** [review]
good work, some changes required though

**Comment by robertkowalski** [comment]
in the beginning the icon has an ugly border always:

Screen Shot 2020-09-10 at 17 05 51

**Comment by robertkowalski** [comment]
Screen Shot 2020-09-10 at 17 06 56


`Last (3/3)` should be named `Finish`

**Comment by robertkowalski** [comment]
there are still issues with theborders in 3 cases, would be good to remove them all, created a video of them

**Comment by robertkowalski** [comment]
Screen Shot 2020-09-29 at 10 53 38

**Comment by robertkowalski** [comment]
Screen Shot 2020-09-29 at 10 53 28

**Comment by robertkowalski** [comment]
the issues are still there.

**Comment by AntonNazarenko** [comment]
@robertkowalski @prdn I've removed this outline from the entire app just checked it in chrome and electron app. Maybe you didn't pull the changes?

**Comment by AntonNazarenko** [comment]
Снимок экрана 2020-09-29 в 12 13 43
Снимок экрана 2020-09-29 в 12 13 54
that's how it looks in the electron app

**Comment by robertkowalski** [comment]
rechecking. i can confirm its fixed, had to rebuild the css

**Review by robertkowalski — APPROVED** [review]
lgtm, will change the texts after merge

**Pull request merged**